modules/aws/ecs_test.go

```diff
@@ -21,3 +23,33 @@ func TestEcsCluster(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "terratest", *c2.ClusterName)
 }
+
+func TestEcsClusterWithInclude(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+
+	client := NewEcsClient(t, region)
+	c1, err := client.CreateCluster(&ecs.CreateClusterInput{
+		ClusterName: aws.String("terratest-extended"),
+		Tags: []*ecs.Tag{&ecs.Tag{
+			Key:   aws.String("test-tag"),
+			Value: aws.String("hello-world"),
+		}},
+	})
+	assert.Nil(t, err)
+
+	defer DeleteEcsCluster(t, region, c1.Cluster)
+
+	assert.Nil(t, err)
+	assert.Equal(t, "terratest-extended", *c1.Cluster.ClusterName)
+
+	c2, err := GetEcsClusterWithIncludeE(t, region, *c1.Cluster.ClusterName, []string{"TAGS"})
+
+	assert.Nil(t, err)
+	assert.Equal(t, "terratest-extended", *c2.ClusterName)
+
+	tag := *c2.Tags[0]
+	assert.Equal(t, "test-tag", *tag.Key)
+	assert.Equal(t, "hello-world", *tag.Value)
```

Comment:
NIT: store the tags you're creating in a var and check for direct equality here to ensure. That'll ensure there aren't _other_ unexpected tags there. Also, it'll avoid an array out of bounds error in case `c2.Tags` is an empty array.